lib/openshift/parser/pod.rb

```diff
@@ -43,10 +43,29 @@ def parse_pod_tags(source_ref, tags)
       end
 
       def parse_containers(pod)
+        container_statuses = pod.status.containerStatuses.index_by { |x| x&.name }
+
+        # TODO we need to do mark&sweep of pods and images togethere, otherwise these container_images will still get
```

Comment:
@agrare so the images created here will still get archived, because they are out of the mark&sweep scope. So we will need to refactor this, so the mark&sweep is done for both pods and images groups.